recipes/sherlockml-boltzmannclean/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sherlockml-boltzmannclean" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "0c028e9f8a67fac787587c20e24ef69792899d1c76e1313515518a9aff0f57ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - pandas
+    - numpy
+    - scipy
+    - scikit-learn
+
+test:
+  imports:
+    - boltzmannclean
+
+about:
+  home: https://sherlockml.com
+  license: Apache 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'Fills missing values in a pandas DataFrame using a Restricted Boltzmann Machine.'
+  description: |
+    Fills missing values in a pandas DataFrame using a Restricted Boltzmann Machine.
+
+extra:
+  recipe-maintainers:
```

Comment:
Please confirm that all of the co-maintainers have agreed to be listed here. If you have not spoken to them in a different forum (either online or in person), please ping them on this PR asking for their permission.

Reply:
confirmed, (I'm sat with them)

Reply:
Great. I'll merge as soon as the CI finishes assuming all of the tests pass.